lib/CarSpeed/CarSpeed.cpp

```diff
@@ -92,10 +102,14 @@ void CarSpeed::task() {
       pid.Compute();
 
       // check range
-      if (output_setpoint < -DAC_MAX)
+      if (output_setpoint < -DAC_MAX) {
+        console << fmt::format("WARN::PID dejustified {} < -{}!\n", output_setpoint, DAC_MAX);
         output_setpoint = -DAC_MAX;
-      if (output_setpoint > DAC_MAX)
+      }
+      if (output_setpoint > DAC_MAX) {
+        console << fmt::format("WARN::PID dejustified {} > {}!\n", output_setpoint, DAC_MAX);
         output_setpoint = DAC_MAX;
```

Comment:
Is this really just a warning? I think it is rather a serious problem where we should brake?

Reply:
let's discuss:
V1) only keep in limits (as implemented)
V2) deactivate constant mode (speed)
V3) stop car

Reply:
The safest option for sure is V3, however, it might even create some risk to simply break (if this happens in a bad moment).
Maybe do V1, but show a warning on the display?

Reply:
I would prefer V2, since only the constant mode is affected.

Reply:
Sure, go with it. However, I would still indicate it to the driver somehow.